metadata/pkg/drivers/utils/utils.go

```diff
@@ -0,0 +1,30 @@
+package utils
```

Comment:
please add headers. You may refer from some other files

Reply:
moved to metadata/pkg/utils